src/nft/components/card/index.tsx

```diff
@@ -80,6 +82,7 @@ export const NftCard = ({
       testId={testId}
       onClick={() => {
         if (bagExpanded) setBagExpanded({ bagExpanded: false })
+        if (sendAnalyticsEvent) sendAnalyticsEvent()
```

Comment:
nit: this can be `sendAnalyticsEvent?.()`, same for bag expanded

Reply:
ooh ty! thank you! I dont think bag expanded can be changed accordingly? setBagExpanded is different from bagExpanded